### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by bgraille**

### CODE REVIEW

File: demo/2D/Karman_vortex_street.py [hunk 1]
```diff
@@ -1,142 +1,273 @@
...
+    scheme_name = 'Geier'
+    xmin, xmax, ymin, ymax = 0., 2., 0., 1.  # bounds of the domain
+    radius = 1./32                           # radius of the obstacle
+    la = 1.                                  # velocity of the scheme
+    rho_o = 1.                               # reference value of the mass
+    u_o = 0.05                               # boundary value of the velocity
+    mu = 5.e-6                               # bulk viscosity
+    zeta = 10*mu                             # shear viscosity
+
+    def moments_choice(scheme_name, mu, zeta):
```
Comment (gouarin) [inline demo/2D/Karman_vortex_street.py]: This function is found in various examples.
I think we can extract it in a separate python file and import it.

File: demo/2D/Karman_vortex_street.py [hunk 2]
```diff
@@ -1,142 +1,273 @@
...
-    qx2 = dummy*qx**2
-    qy2 = dummy*qy**2
-    q2  = qx2+qy2
-    qxy = dummy*qx*qy
-
-    dico = {
-        'box':{'x':[xmin, xmax], 'y':[ymin, ymax], 'label':[0, 1, 0, 0]},
-        'elements':[pylbm.Circle([.3, 0.5*(ymin+ymax)+2*dx], radius, label=2)],
-        'space_step':dx,
+    scheme_name = 'Geier'
```
Comment (gouarin) [inline demo/2D/Karman_vortex_street.py]: Add in a comment the other possibilities ('dHumiere', 'Geier' or 'Lallemand')

File: demo/2D/Kelvin_Helmoltz.py [hunk 3]
```diff
@@ -1,148 +1,257 @@
...
+    -------
+
+    sol
+        <class 'pylbm.simulation.Simulation'>
+
     """
-    rhoo = 1.
-    U = .5
+    # parameters
+    scheme_name = 'Geier'
```
Comment (gouarin) [inline demo/2D/Kelvin_Helmoltz.py]: Add in a comment the other possibilities ('dHumiere', 'Geier' or 'Lallemand')

File: demo/2D/bend.py [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+
+    Returns
+    -------
+
+    sol
+        <class 'pylbm.simulation.Simulation'>
+
+    """
+    # parameters
+    scheme_name = 'Geier'
```
Comment (gouarin) [inline demo/2D/bend.py]: Add in a comment the other possibilities ('dHumiere', 'Geier' or 'Lallemand')

File: demo/2D/step.py [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+
+    Returns
+    -------
+
+    sol
+        <class 'pylbm.simulation.Simulation'>
+
+    """
+    # parameters
+    scheme_name = 'Geier'
```
Comment (gouarin) [inline demo/2D/step.py]: Add in a comment the other possibilities ('dHumiere', 'Geier' or 'Lallemand')

File: pylbm/simulation.py [hunk 6]
```diff
@@ -11,10 +11,10 @@
 import sys
 import logging
 import types
-from textwrap import dedent
-from six import string_types
+# from textwrap import dedent
```
Comment (gouarin) [inline pylbm/simulation.py]: remove these lines if these packages are finally not used.

**Comment by bgraille** [comment]
I will propose other examples in a future PR but we actually need the added features (stability and viewer)

**Pull request merged**